### DISCUSSION TITLE
How to customize the layout

### DISCUSSION THREAD

**Discussion by tomli-123** [vercel/next.js#67321] [App Router]
How to customize the layout of individual pages after setting up the global layout (e.g. detail pages without page headers) in app router mode

**Comment by SouthLink** [comment]
You can create multiple root layouts and use this feature through routing groups to make your global layouts appear in groups.
[Creating multiple root layouts](https://nextjs.org/docs/app/building-your-application/routing/route-groups#creating-multiple-root-layouts)
